engine/src/main/java/org/entando/entando/web/user/UserController.java

```diff
@@ -163,6 +165,15 @@ public void setUserValidator(UserValidator userValidator) {
         return new ResponseEntity<>(new RestResponse(result), HttpStatus.OK);
     }
 
+    private BindingResult createBindingResult(@PathVariable String username) {
+        Map<String, String> map = new HashMap<>();
+        map.put("username", username);
+        BindingResult bindingResult = new MapBindingResult(map, "username");
+        ObjectError error = new ObjectError("username", "Sorry. You can't delete the administrator user");
```

Comment:
you have to use message.properties file for text messages

Reply:
Fixed